pkgs/tools/security/bloodhound-py/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "bloodhound-py";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "fox-it";
+    repo = "BloodHound.py";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-K74VntcRcIgKu48JhLuWeFcqPQg7vurd0+H1e6Kk3Rg=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    impacket
+    ldap3
+    dnspython
+  ];
+
+  doCheck = false;
```

Comment:
there should be a comment explaining why

Reply:
I think other distributions like blackarch does not execute the tests of bloodhound.py.

Reply:
I think this package does not provide tests in the first place.

Reply:
Right, I just checked the repository and it does not contain any tests.